packages/fx-core/templates/plugins/resource/cicd/azdo/README.md

```diff
@@ -0,0 +1,47 @@
+# How to use pre-cooked pipelines on Azure DevOps
+
+## Prerequisites
+- Teams app projects are version controlled by Azure DevOps.
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
+- An Azure service princial with necessary permissions.
+
+## Steps
+After the pre-cooked pipelines are scaffolded successfully, the following steps are expected to be performed:
+1. Add the scaffolded yml files into Azure DevOps by `git add`,`git commit` and `git push`.
```

Comment:
Commit and push your project repo to github remote repo, including the CICD yml files.